src/app/file-browser/components/edit-tags/edit-tags.component.html

```diff
@@ -19,6 +22,10 @@
       </div>
     </div>
   </div>
+  <p>
+    <span class="manage-tags-link" (click)="onManageTagsClick()">
+      Manage {{this.tagType === 'keyword' ? "tags" : 'custom metadata'}}</span> in archive settings.
+  </p>
```

Comment:
Currently this link goes to the existing manage tags modal, which for the moment is the place to manage both tags and custom metadata. I expect it will make sense to start differentiating where it goes as part of https://permanent.atlassian.net/browse/PER-9023

Reply:
Hm... my interpretation of the design is that this link should only appear when the tag editor is actually open (otherwise it takes up a lot of space, especially with two links being present). But I think this is ambiguous; I can't actually find a design where the tag editors are closed. We should probably run this by the experience team to see what they think.

Also, a slight nitpick but I there should be a tiny bit of margin between this link and what is above it. It looks slightly off compared to the design since it seems a bit to close to the tag editor. (Though again, the exp team can approve/disapprove how it looks here!)